### PR TITLE
[10.0][MIG] stock_picking_back2draft

### PR DESCRIPTION
**Pull request by Saphery**
Migration of stock_picking_back2draft to v10

### CODE REVIEW

**Comment by Saphery** [comment]
The CLA is signed and sent. Please check it

**Review by HaraldPanten — COMMENTED** [review]
Hi @Saphery , I think you could use these modules:

https://github.com/OCA/stock-logistics-workflow/tree/10.0/stock_cancel

+

https://github.com/OCA/stock-logistics-workflow/tree/10.0/stock_cancel_delivery

instead of migrating stock_picking_back2draft to v10. 

What do you think?

**Comment by Saphery** [comment]
Hi @HaraldPanten 

I have reviewed these modules. 
But they don't do exactly the same.
Back2draft transforms the canceled stock pickings in draft, and stock_cancel only does it for the stock picking in the "done" state.

I need the first case.

**Review by HaraldPanten — APPROVED** [review]
You're right @Saphery 

Then let me give you my feedback about the module.

It functionally works :+1:

**Comment by tafaRU** [comment]
Please preserve commit history following technical method explained in https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-10.0

**Comment by Saphery** [comment]
@tafaRU It's done

File: stock_picking_back2draft/models/stock.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import _, models, api
+from odoo.exceptions import Warning as UserError
```
Comment (rousseldenis) [inline stock_picking_back2draft/models/stock.py]: You can import UserError directly

File: stock_picking_back2draft/tests/test_picking.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# © 2016 Lorenzo Battistini - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo.tests.common import TransactionCase
+from odoo.exceptions import Warning as UserError
```
Comment (rousseldenis) [inline stock_picking_back2draft/tests/test_picking.py]: You can import UserError directly

**Review by rousseldenis — CHANGES_REQUESTED** [review]
Code review. Minor comments. LGTM

**Comment by rousseldenis** [comment]
Please squash all Transbot commits into one

**Comment by Saphery** [comment]
@rousseldenis Minor fixes and squash transbot commits done

**Review by rousseldenis — APPROVED** [review]

**Review by Jortolsa-S73 — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]

**Pull request merged**